src/Microsoft.Tye.Core/CoreStrings.resx

```diff
@@ -153,6 +153,9 @@
   <data name="MustBeAnInteger" xml:space="preserve">
     <value>"{value}" value must be an integer.</value>
   </data>
+  <data name="MustBeAnIPAddress" xml:space="preserve">
+    <value>"{value}" value must be an IP address, * or localhost.</value>
```

Comment:
Nit: add quotes around `*` and `localhost`?

Reply:
Done